x/staking/simulation/operations_test.go

```diff
@@ -381,7 +381,7 @@ func setupValidatorRewards(app *simapp.SimApp, ctx sdk.Context, valAddress sdk.V
 	decCoins := sdk.DecCoins{sdk.NewDecCoinFromDec(sdk.DefaultBondDenom, math.LegacyOneDec())}
```

Comment:
(not this file, but this package):

Any reasons why you moved `decoder_test` and `operations_test`, instead of converting them to use depinject like in [slashing or bank](https://github.com/cosmos/cosmos-sdk/pull/12897)?

Reply:
I'll take a look at that, I might have checked how it was done there before that PR. Gonna check and hopefully get those done with mocks too